src/vaadin-login-form.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="vaadin-login-form-wrapper.html">
+<link rel="import" href="../../vaadin-text-field/src/vaadin-text-field.html">
+<link rel="import" href="../../vaadin-text-field/src/vaadin-password-field.html">
+
+<dom-module id="vaadin-login-form">
+  <template>
+      <style>
+        #loginForm * {
+          width: 100%;
+        }
+      </style>
+      <vaadin-login-form-wrapper
+        theme$="[[theme]]"
+        id="loginFormWrapper"
+        action="{{action}}"
+        disabled="{{disabled}}"
+        error="{{error}}"
+        no-forgot-password="{{noForgotPassword}}"
+        i18n="{{i18n}}"
+        on-login="_handleEvent"
+        on-forgot-password="_handleEvent">
+
+        <form id="loginForm" method="POST" action$="[[action]]" slot="form">
+          <vaadin-text-field name="username" label="[[i18n.form.username]]"
+              id="username" required on-keydown="_handleInputKeydown"
+              autocapitalize="none" autocorrect="off" spellcheck="false">
+            <input type="text" slot="input">
+          </vaadin-text-field>
+
+          <vaadin-password-field name="password" label="[[i18n.form.password]]" id="password" required on-keydown="_handleInputKeydown"
+              spellcheck="false">
+            <input type="password" slot="input">
+          </vaadin-password-field>
+
+          <vaadin-button part="vaadin-login-submit" theme="primary contained" on-click="submit" disabled$="[[disabled]]">[[i18n.form.submit]]</vaadin-button>
+        </form>
+      </vaadin-login-form>
+  </template>
+
+  <script>
+    (function() {
+      /**
+       * `<vaadin-login-form>` is a Web Component providing an easy way to require users
+       * to log in into an application. Note that component has no shadowRoot.
+       *
+       * ```
+       * <vaadin-login-form></vaadin-login-form>
+       * ```
+       *
+       * Component has to be accessible from the `document` layer in order to allow password managers to work properly with form values.
+       * Using `<vaadin-login-overlay>` allows to always attach the component to the document root.
```

Comment:
nit:  would be nice to mention body here, e.g. "document body" sounds better than "document root"